python-sdk/example_dags/example_bigquery_dynamic_map_task.py

```diff
@@ -18,9 +18,15 @@
 from astro.table import Metadata
 
 ASTRO_BIGQUERY_DATASET = os.getenv("ASTRO_BIGQUERY_DATASET", "dag_authoring")
-ASTRO_GCP_CONN_ID = os.getenv("ASTRO_GCP_CONN_ID", "google_cloud_default")
+ASTRO_GCP_CONN_ID = os.getenv("ASTRO_GCP_CONN_ID", "google_cloud_platform")
```

Comment:
Why are we moving out from the `google_cloud_default` default...? If we need to this to have another value due to the Astro deployment, we can always define the environment variable `ASTRO_GCP_CONN_ID` in those scenarios.